src/bosh_azure_cpi/spec/unit/vm_manager/create/shared_stuff.rb

```diff
@@ -111,14 +121,42 @@
   end
   let(:load_balancer) do
     {
-      name: 'fake-lb-name'
+      name: 'fake-lb-name',
+      backend_address_pools: [
+        {
+          name: 'fake-pool-name',
+          id: 'fake-pool-id',
+          provisioning_state: 'fake-pool-state',
+          backend_ip_configurations: []
+        }
+      ]
     }
   end
   let(:application_gateway) do
     {
-      name: 'fake-ag-name'
+      name: 'fake-ag-name',
+      backend_address_pools: [
+        {
+          name: 'fake-pool-name',
+          id: 'fake-pool-id',
+          provisioning_state: 'fake-pool-state',
+          backend_ip_configurations: []
+        }
+      ]
     }
   end
+  # let(:network_interfaces) do
```

Comment:
Why this commented out code is there?

Reply:
A number of existing specs have `let(:network_interfaces)` declarations. I was playing around with adding one here in the shared `let`s to consolidate (at least some of) these duplicated code sections, but I didn't complete that change.

I can remove this section, or leave it as a reminder of an opportunity for future refactoring, as you prefer.

Reply:
I you would like to complete this please do that. Otherwise I would suggest to remove the commented out code.

Reply:
Done.